src/AppBundle/Form/DataTransformer/TagsToCollectionTransformer.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: aragorn
+ * Date: 06/06/18
+ * Time: 14:58
+ */
+
+namespace AppBundle\Form\DataTransformer;
+
+use AppBundle\Entity\Tag;
+use Doctrine\Common\Persistence\ObjectManager;
+use Symfony\Component\Form\DataTransformerInterface;
+use Doctrine\Common\Collections\ArrayCollection;
+
+class TagsToCollectionTransformer implements DataTransformerInterface
+{
+    /**
+     * @var ObjectManager
+     */
+    private $manager;
+
+    public function __construct(ObjectManager $manager)
+    {
+        $this->manager = $manager;
```

Comment:
Il ne faut pas envoyer tout le manager en paramètre mais seul le service dont tu as besoin. Ici à priori c'est le Repository des Tags. Tu peux te référer à la PSR 11 pour comprendre pourquoi éviter de faire cela : 

https://www.php-fig.org/psr/psr-11/meta/#4-recommended-usage-container-psr-and-the-service-locator